[FEATURE] Create tmp-def-ia-supervise.json

JTEPIX opened the pull request:
## 🔆 Problème

<!-- Décrivez ici le besoin ou l'intention couvert par cette Pull Request. -->

## ⛱️ Proposition

<!-- Ajoutez à cet endroit, si nécessaire, des détails concernant la solution technique retenue et mise en oeuvre, des difficultés ou problèmes rencontrés. -->

## 🌊 Remarques

<!-- Des infos supplémentaires, trucs et astuces ? -->

## 🏄 Pour tester

https://app-pr13137.review.pix.fr/modules/tmp-ia-def-ind/details

JTEPIX committed:
Create tmp-def-ia-supervise.json

pix-bot-github commented:
Choisir les applications à déployer :

- [x] [API](https://api-pr13137.review.pix.fr/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-review-pr13137/environment) <!-- pix-api-review -->
- [x] [App (.fr)](https://app-pr13137.review.pix.fr) – [App (.org)](https://app-pr13137.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-app-review-pr13137/environment) <!-- pix-app-review -->
- [ ] [Orga (.fr)](https://orga-pr13137.review.pix.fr) – [Orga (.org)](https://orga-pr13137.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-orga-review-pr13137/environment) <!-- pix-orga-review -->
- [ ] [Certif (.fr)](https://certif-pr13137.review.pix.fr) – [Certif (.org)](https://certif-pr13137.review.pix.org) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-certif-review-pr13137/environment) <!-- pix-certif-review -->
- [ ] [Junior](https://junior-pr13137.review.pix.fr) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-junior-review-pr13137/environment) <!-- pix-junior-review -->
- [ ] [Admin](https://admin-pr13137.review.pix.fr) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-admin-review-pr13137/environment) <!-- pix-admin-review -->
- [ ] [API MaDDo](https://pix-api-maddo-review-pr13137.osc-fr1.scalingo.io/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-api-maddo-review-pr13137/environment) <!-- pix-api-maddo-review -->
- [ ] [Audit Logger](https://pix-audit-logger-review-pr13137.osc-fr1.scalingo.io/api/) | [👩‍💻 Dashboard Scalingo](https://dashboard.scalingo.com/apps/osc-fr1/pix-audit-logger-review-pr13137/environment) <!-- pix-audit-logger-review -->

> [!IMPORTANT]
> N'oubliez pas de déployer l'API pour pouvoir accéder aux fronts et/ou à l’API MaDDo.

JTEPIX committed:
Update tmp-def-ia-supervise.json

JTEPIX committed:
Update tmp-def-ia-supervise.json

Natikonicoff committed:
conversation arthur lila

Natikonicoff committed:
transition avant lecon 1

Natikonicoff committed:
lecon 1

Natikonicoff committed:
Lecon 1

Natikonicoff committed:
emojis

Natikonicoff committed:
modifs texte

Natikonicoff committed:
suppression nsbp parasite

Natikonicoff committed:
strong

Natikonicoff committed:
lessons2

Natikonicoff committed:
lesson3

Natikonicoff committed:
Update tmp-def-ia-supervise.json

JTEPIX committed:
Update tmp-def-ia-supervise.json

modification des POI et ajout d'un step sur la décomposition de l'image en valeur numérique

Natikonicoff committed:
lesson2

Natikonicoff committed:
supression element vide

Natikonicoff committed:
deux dernieres lessons modififies

Natikonicoff committed:
coquille

Natikonicoff committed:
ajout step lesson 4

Natikonicoff committed:
apprentissage auto ajoute

pix-service commented:
:tada: This PR is included in version 5.182.0 :tada:

The release is available on [GitHub release](https://github.com/1024pix/pix/releases/tag/v5.182.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
